website/pages/en/new-chain-integration.mdx

```diff
@@ -1,75 +1,110 @@
 ---
-title: Integrating New Networks
+title: New Chain Integration
 ---
 
-Graph Node can currently index data from the following chain types:
+Chains can bring subgraph support to their ecosystem by starting a new `graph-node` integration. Subgraphs are a powerful indexing tool opening a world of possibilities for developers. Graph Node already indexes data from the chains listed here. If you are interested in a new integration, there are 2 integration strategies:
 
-- Ethereum, via EVM JSON-RPC and [Ethereum Firehose](https://github.com/streamingfast/firehose-ethereum)
-- NEAR, via a [NEAR Firehose](https://github.com/streamingfast/near-firehose-indexer)
-- Cosmos, via a [Cosmos Firehose](https://github.com/graphprotocol/firehose-cosmos)
-- Arweave, via an [Arweave Firehose](https://github.com/graphprotocol/firehose-arweave)
+1. **EVM JSON-RPC**
+2. **Firehose**: All Firehose integration solutions include Substreams, a large-scale streaming engine based off Firehose with native `graph-node` support, allowing for parallelized transforms.
 
-If you are interested in any of those chains, integration is a matter of Graph Node configuration and testing.
+> Note that while the recommended approach is to develop a new Firehose for all new chains, it is a requirement for non-EVM chains.
```

Comment:
Nitpick:
```suggestion
> Note that while the recommended approach is to develop a new Firehose for all new chains, it is only a requirement for non-EVM chains.
```

Reply:
or even:
```suggestion
> Note that while the recommended approach is to develop a new Firehose for all new chains, it is not a required for EVM chains.
```

Reply:
(basically, I feel like there's a tiny bit of ambiguity with the sentence's logic as it is)